datasets/new1/WD-new1-0.txt

```diff
@@ -0,0 +1,23 @@
+WEEK_DATA
+new1
+
+REQUIREMENTS
+S0 HeadNurse (1,1) (0,0) (0,0) (0,0) (0,0) (1,1) (0,0)
+S0 Nurse (1,2) (1,1) (1,1) (0,1) (1,1) (1,1) (0,1)
+S1 HeadNurse (1,1) (0,1) (1,1) (0,0) (0,0) (0,0) (0,0)
+S1 Nurse (1,1) (1,1) (0,1) (0,1) (1,1) (1,1) (1,1)
+S2 HeadNurse (0,0) (1,1) (0,0) (0,0) (1,1) (1,1) (0,0)
+S2 Nurse (0,1) (1,1) (1,1) (1,1) (1,1) (0,1) (1,1)
+S4 HeadNurse (0,0) (1,1) (0,0) (0,0) (1,1) (0,1) (0,0)
+S4 Nurse (0,1) (0,0) (0,0) (1,1) (1,1) (0,1) (0,1)
+S5 HeadNurse (0,0) (0,1) (0,0) (0,0) (0,1) (0,1) (0,0)
+S5 Nurse (0,1) (0,1) (0,1) (0,0) (0,0) (0,1) (1,1)
+
+SHIFT_OFF_REQUESTS = 3
```

Comment:
Does it not make more sense to have requests on shift types?

Reply:
Yes, we could create a new issue to be able to handle both I think, and even at the same time.

Reply:
Done, I just added the issue.